chip8_vm/README.md

```diff
@@ -0,0 +1,37 @@
+Virtual Machine for the CHIP-8 programming language
+==
+This crate implements a virtual machine of the CHIP-8 programming language. It
+can be used as the back end for CHIP-8 emulator projects / debuggers etc.
+
+Status
+==
+* All 35 original Chip-8 instructions are implemented.
+
+Usage
+==
+This crate only has no system library dependencies, only dependencies on other
+rust crates. It can be used as a back end for your own emulator by adding it as
+a dependency in your `Cargo.toml`.
+
+To depend on the latest released version check [crates.io] (https://crates.io/crates/chip8_vm)
+
+To depend on the development version you can add the follwing to your
+`Cargo.toml`
+
+```toml
+[dependencies.chip8_vm]
+git = "https://github.com/jakerr/chip8-rust"
+```
+
+See an exmaple integration with a UI in the [chip8_ui](https://github.com/jakerr/chip8-rust/blob/master/src/main.rs) crate.
+
+Spec
+==
+These two resources were used as the spec for this vm:
+* [Cowgod's Chip-8 Technical Reference v1.0](http://devernay.free.fr/hacks/chip8/C8TECH10.HTM)
+* [Mastering Chip-8 By Matthew Mikolay](http://mattmik.com/chip8.html)
+They were both incredibly helpful so thanks to the authors!
```

Comment:
Potentially add a newline so the note does not display right after the second bullet point.